stage3/searchwing-antennabox/rootfs/home/searchwing/srtm_downloader.py

```diff
@@ -0,0 +1,49 @@
+import json
+import math
+from tqdm import tqdm
```

Comment:
Do we actually need the tqdm progressbar? If we need it I guess we have to install it somewhere.